### PR TITLE
Bg image issue fix

### LINKED ISSUE #219: Calciyum page LH issue.
@asthabh23 [/en/products/calciyum](https://215-fullwidth-links--fonterra-fernleaf--hlxsites.hlx.page/en/products/calciyum) page is failing LH score, CLS problem is caused by the background image which we will track in a different issue.

_Originally posted by @sselvara in https://github.com/hlxsites/fonterra-fernleaf/issues/216#issuecomment-1699580375_

### PR DESCRIPTION
**Pull request by ParthasharathiB**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #219, #214

Test URLs:
- Before: https://main--fonterra-fernleaf--hlxsites.hlx.page/en
- After: https://219-calciyum-bgimg--fonterra-fernleaf--hlxsites.hlx.page/en/products/calciyum

-Impact pages : 
https://219-calciyum-bgimg--fonterra-fernleaf--hlxsites.hlx.live/en/products/calciyum
https://219-calciyum-bgimg--fonterra-fernleaf--hlxsites.hlx.live/en/recipes
https://219-calciyum-bgimg--fonterra-fernleaf--hlxsites.hlx.live/en/stories-and-tips
https://219-calciyum-bgimg--fonterra-fernleaf--hlxsites.hlx.live/en/stories-and-tips/5-snack-hacks-for-your-school-aged-child

### CODE REVIEW

**Comment by asthabh23** [comment]
I see CLS on stories-and-tips page:
https://pagespeed.web.dev/analysis/https-219-calciyum-bgimg--fonterra-fernleaf--hlxsites-hlx-live-en-stories-and-tips/c15715rqbc?form_factor=mobile

**Comment by sselvara** [comment]
@ParthasharathiB Mobile performance is dropped for calciyum page https://pagespeed.web.dev/analysis/https-219-calciyum-bgimg--fonterra-fernleaf--hlxsites-hlx-live-en-products-calciyum/zo1xk8ch2r?form_factor=mobile.

**Comment by ParthasharathiB** [comment]
@sselvara  can you please check again i can see 100% https://pagespeed.web.dev/analysis/https-219-calciyum-bgimg--fonterra-fernleaf--hlxsites-hlx-live-en-products-calciyum/zcqfnxlm3e?form_factor=mobile

File: blocks/story-list/story-list.js [hunk 1]
```diff
@@ -40,6 +40,8 @@ export default async function decorate(block) {
   } else {
     block.append('no result found');
   }
-  const boundFunction = new ProcessStoriesBgImage().updateStoriesBgImage.bind(this, bgConfigParams);
-  boundFunction();
+  setTimeout(() => {
```
Comment (sselvara) [inline blocks/story-list/story-list.js:43]: use same syntax window.setTimeout

**Review by sselvara — CHANGES_REQUESTED** [review]

**Review by sselvara — APPROVED** [review]

**Pull request merged**